simulation_parameters/microbe_stage/organelles.json

```diff
@@ -719,6 +719,40 @@
         "editorButtonGroup": "External",
         "editorButtonOrder": 3
     },
+    "slimeJet": {
+        "mpCost": 60,
+        "initialComposition": {
+            "ammonia": 1,
+            "phosphates": 1
+        },
+        "hexes": [
+            {
+                "r": 0,
+                "q": 0
+            }
+        ],
+        "processes": {
+            "mucilage_synthesis": 1
+        },
+        "components": {
+            "processor": {},
+            "storage": {
+                "capacity": 0.5
+            },
+            "slimeJet": {}
+        },
+        "prokaryoteChance": 0.5,
+        "chanceToCreate": 0.5,
```

Comment:
I think the chance to create should be set to be pretty low as the AI will find these pretty useless currently, right?

Reply:
Reading the rest of the code I see you did address that pretty comprehensively, still I think the slime jets should maybe be a bit more of a specialist organelle so maybe reducing this just a bit to "0.4" or "0.3" would still be good.